src/shadow-embed.js

```diff
@@ -127,7 +127,12 @@ function createShadowRootPolyfill(hostElement) {
     // TODO(@dvoytenko) Consider to switch to a type union instead.
     /** @type {?}  */ (doc.createElement('i-amphtml-shadow-root')));
   hostElement.appendChild(shadowRoot);
-  hostElement.shadowRoot = hostElement.__AMP_SHADOW_ROOT = shadowRoot;
+  hostElement.__AMP_SHADOW_ROOT = shadowRoot;
+  Object.defineProperty(hostElement, 'shadowRoot', {
+    enumerable: true,
+    configurable: true,
```

Comment:
Is there a test that validates the `shadowRoot` is configurable and enumerable? Does behaviour rely on these changes from the defaults of `defineProperty`?

Reply:
@jridgewell ^^

Reply:
This matches the real implementations:

```js
Object.getOwnPropertyDescriptor(document.body.__proto__.__proto__.__proto__, 'shadowRoot')
// {get: ƒ, set: undefined, enumerable: true, configurable: true}